coordinator/src/lib.rs

```diff
@@ -270,7 +275,14 @@ impl Initializer for Coordinator {
 
         let (validator_set, params) = services.init_consensus.init_consensus(session_id);
 
-        self.max_body_size.set(params.max_body_size() as usize).expect("this must be the first assignment");
+        // FIXME: Should we cache max body size?
```

Comment:
I think we can safely remove the caching part. What you stated your concern on it on discord seems the right way to go. I understand `max_body_size` should be a parameter that can be changed over time.

Reply:
Thanks! I'll do it.